io.neovim.nvim.yaml

```diff
@@ -5,7 +5,9 @@ runtime: org.freedesktop.Sdk
 runtime-version: '20.08'
 sdk: org.freedesktop.Sdk
 
-command: nvim
+command: /app/nvim.sh
+
+desktop-file-name-suffix: ' (Flatpak)'
```

Comment:
```suggestion
```

this is not the common practice to do that.

Reply:
When I attempted to change the build by changing the `-D CMAKE_INSTALL_PREFIX=/app/neovim/` the build would continually fail. So I instead dropped the shim script into the Flatpak, and pointed the command to it. So when launched from the command-line, it functions as intended. Also, the `desktop-file-name-suffix` makes it so you know that the Neovim you installed is actually a flatpak. Something I continually find not knowing.

Reply:
I was really just talking about that single line. We don't add suffixes (or prefixes) to desktop file for flatpak...

Just to mark "beta" from the beta branch.

Reply:
Is there a way to identify that this is a Flaptak rather than a package manager installed package?

Reply:
> I was really just talking about that single line. We don't add suffixes (or prefixes) to desktop file for flatpak...
> 
> Just to mark "beta" from the beta branch.

This has been removed for this PR.

Reply:
> Is there a way to identify that this is a Flaptak rather than a package manager installed package?

That’s not our job. It should ideally be up for the DE; we don’t want a Flatpak-centric distro to have " (Flatpak)" after every app.